### PR TITLE
Installer cleanup

### LINKED ISSUE #7: Update/uninstall leaves VS2013 extension behind
If you uninstall code contracts, either manually or by installing a new version, the VS2013 extension stays installed or stays at the old version. The whole folder C:\Program Files (x86)\Microsoft Visual Studio 12.0\Common7\IDE\Extensions\Microsoft\CodeContracts isn't removed.

### LINKED ISSUE #8: Installer creates a C:\VS11Root folder
Whenever I install Code Contracts, the installer creates a C:\VS11Root folder that seems to contain the Visual Studio extension. This happens with machines where VS2010 or VS2013 is installed (haven't tested VS2012).

### PR DESCRIPTION
**Pull request by sharwell**
* Primary changes
  * Use newer WiX binaries from NuGet
  * Fix #7
  * Fix #8
* Secondary changes
  * Remove `Feature_VSTSCheck`
  * Use variables provided by WiX and remove **dev10.wxs**, **dev12.wxs**, **dev14.wxs**

:memo: Users with previous versions installed for 2013 and/or 2015 will have to *manually remove* the old installation folders via the following steps.

1. Uninstall Code Contracts via add/remove programs
1. Delete the following folders if they exist on your machine:
  1. Code Contracts for Visual Studio 2013:

        ```
        C:\Program Files (x86)\Microsoft Visual Studio 12.0\Common7\IDE\Extensions\Microsoft\CodeContracts
        ```
  1. Code Contracts for Visual Studio 2015:

        ```
        C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\Extensions\Microsoft\CodeContracts
        ```
1. Install the new version of Code Contracts

In the future, upgrades will not leave these stray folders behind.

### CODE REVIEW

File: Microsoft.Research/ManagedContract.Setup/ManagedContracts.wxs [hunk 1]
```diff
@@ -210,43 +208,15 @@
...
-    <PropertyRef Id="VS10_ROOT_FOLDER"/>
-    <PropertyRef Id="VS11_ROOT_FOLDER"/>
-    <PropertyRef Id="VS12_ROOT_FOLDER_EXISTS"/>
-    <PropertyRef Id="VS14_ROOT_FOLDER_EXISTS"/>
+    <PropertyRef Id="VS2010DEVENV"/>
+    <!--<PropertyRef Id="VS10EXPRESSDEVENV"/>-->
+    <PropertyRef Id="VS2010_ROOT_FOLDER"/>
+    <PropertyRef Id="VS2012_ROOT_FOLDER"/>
+    <PropertyRef Id="VS2013_ROOT_FOLDER"/>
+    <PropertyRef Id="VS14_ROOT_FOLDER"/>
```
Comment (sharwell) [inline Microsoft.Research/ManagedContract.Setup/ManagedContracts.wxs]: :memo: In later releases of WiX 3.10, this name changes to `VS2015_ROOT_FOLDER`. It's not available in the current build from NuGet, but the value here works just as well.

File: Microsoft.Research/ManagedContract.Setup/ManagedContracts.wxs [hunk 2]
```diff
@@ -1431,63 +1451,8 @@
     -->
     <CustomActionRef Id="VS2012Setup" />
     <CustomActionRef Id="VS2013Setup" />
-    <CustomActionRef Id="VS2015Setup" />
+    <CustomActionRef Id="VS14Setup" />
```
Comment (SergeyTeplyakov) [inline Microsoft.Research/ManagedContract.Setup/ManagedContracts.wxs]: Orh... what an inconsistency:))

**Comment by SergeyTeplyakov** [comment]
I'm not an expert in this area, but changes are looking good to me! :+1:

File: Microsoft.Research/ManagedContract.Setup/ManagedContracts.wxs (hunk 2 above)
Comment (sharwell) [inline Microsoft.Research/ManagedContract.Setup/ManagedContracts.wxs]: This is the same as the note above.

**Comment by SergeyTeplyakov** [comment]
Merged! :clap:

**Pull request merged**